plugin/src/main/java/com/iridium/iridiumskyblock/commands/WithdrawCommand.java

```diff
@@ -52,6 +54,8 @@ public void execute(CommandSender sender, String[] args) {
             Optional<BankItem> bankItem = IridiumSkyblock.getInstance().getBankItemList().stream().filter(item -> item.getName().equalsIgnoreCase(args[1])).findFirst();
             if (bankItem.isPresent()) {
                 bankItem.get().withdraw(player, Double.parseDouble(args[2]));
+                IridiumSkyblock.getInstance().getDatabaseManager().getIslandTransactionTableManager().getEntries().add(
+                        new BankTransaction(island.get(), bankItem.get().getName(), Double.parseDouble(args[2]), System.currentTimeMillis(), player.getUniqueId(), TransactionType.WITHDRAW));
```

Comment:
Also its important to note that if I do /is withdraw money 10 it might not actually withdraw 10 money, if I only have 5 money in the bank it will withdraw 5, but the transaction will still show as 10

Reply:
oh help me for this